datageneration/retrieve_combinations.py

```diff
@@ -212,7 +215,11 @@ def fetch_examples_recursively(curr_page, fetched_examples):
                 for _example in example.split(';'):
                     if len(fetched_examples) > num_examples - 1:
                         return fetched_examples
-                    if is_roman(_example):
+
+                    if not add_non_roman_examples:
+                        if is_roman(_example):
+                            fetched_examples.add(_example)
+                    else:
```

Comment:
This will not guarantee that a non-roman example is returned right? Should it maybe better be "if not is_roman(_example):" to force it to use another alphabet?

Reply:
Do you mean condition "else" can't guarantee?

Reply:
we will write a function that counts non-roman alphabets once we create the daasets